src/components/ui/Input/InputWrapper.tsx

```diff
@@ -2,23 +2,23 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import React, { forwardRef, ReactNode, Ref } from 'react';
+import { BaseProps } from '../Base';
 
 import { Size } from './';
 import { StyledInputWrapper } from './Styled';
 
-export interface InputWrapperProps {
+export interface InputWrapperProps extends BaseProps {
   leadingIcon?: ReactNode;
   sizing?: Size;
-  className?: string;
   children?: ReactNode | ReactNode[];
 }
 
 export const InputWrapper = forwardRef(
   (props: InputWrapperProps, ref: Ref<HTMLSpanElement>) => {
-    const { leadingIcon, children } = props;
+    const { leadingIcon, children, sizing, ...rest } = props;
 
     return (
-      <StyledInputWrapper ref={ref} {...props} data-testid="input-wrapper">
+      <StyledInputWrapper ref={ref} sizing={sizing} leadingIcon={leadingIcon} {...rest} data-testid="input-wrapper">
```

Comment:
Was this sizing prop not being used before?

Reply:
It was being used before, but it was being sent as part of ...props

you can think of ...props as an object containing "the remaining props". 

So sizing={sizing}, leadingIcon={leadingIcon}, and {...rest} is kind of the same, but I've separated out the sizing and leading icon prop from the ...rest object (same as ...props).